netbox/netbox/constants.py

```diff
@@ -1,3 +1,18 @@
+CORE_APPS = (
```

Comment:
Should probably put a doc string here for what this is / what effect putting an app here has, sort of magic right now

Reply:
It's a list of the core apps; not sure how to expand on that. I'd expect that usage inspection would make clear its purpose if not already apparent.